### ISSUE TITLE
BOTW moon jump broken

### ISSUE THREAD

**Issue by MasterBubbles** [WerWolv/EdiZon_CheatsConfigsAndScripts#113]
I have a problem with the moon jump (R+B) on breath of the wild, it works for a while then stops working, I have to save and re-load the game for it to start working again. 4x speed does not seem to be working either. Except that all other cheats work fine. I don't care much about the 4x speed, but is there any way to make the moon jump work without having to reload the game?

**Comment by rondhi** [comment]
Try talking to an NPC. That fixes it for me most of the time.

**Comment by MasterBubbles** [comment]
Thanks, it seems to be working but not with all NPCs, I guess in some situations the best is simply to save / re-load the game

**Comment by rondhi** [comment]
Yeah, it doesn't seem to work with NPCs. Sometimes it's NPCs that give you a choice.

A trick I used was to use a save editor to unlock all koroks. Since there's 900 of them, it's easy to find one while out in the overworld.

**Comment by MasterBubbles** [comment]
So, adding 900 koroks in the inventory though a save editor causes the moon jump to always be activated?
